nixos/modules/security/sudo-rs.nix

```diff
@@ -76,15 +73,15 @@ in
       '';
     };
 
-    extraRules = mkOption {
+    extraRules = lib.mkOption {
       description = ''
         Define specific rules to be in the {file}`sudoers` file.
         More specific rules should come after more general ones in order to
-        yield the expected behavior. You can use mkBefore/mkAfter to ensure
+        yield the expected behavior. You can use lib.mkBefore/mkAfter to ensure
```

Comment:
revert

Reply:
really? maybe even add `lib.mkAfter`?

Reply:
Put it in backticks (`lib.mkBefore`/`lib.mkAfter`) and you have yourself a deal.